.gitignore

```diff
@@ -16,3 +16,4 @@ overline.*
 outputs/rnet/metadata.json
 outputs/routes/metadata.json
 outputdata/
+outputs/zones/metadata.json
```

Comment:
Why exclude the metadata? Worth publishing in 'inputdata' if too big to commit or you don't want to share for other reasons. Important for reproducibility I think.

Reply:
It's not useful, just gibberish